README.md

```diff
@@ -40,31 +40,38 @@ a lot of resources.
     ┌───────────────────┐
     │ Testing Framework ├────┐
     └──────┬────────────┘    │ Management over SSH
-           │                 ├────────────────────┐
-           │          ┌──────┴─────┐        ┌─────┴──────┐
-           │          │ TempestaFW │        │ Web Server │
-           │          └──────┬─────┘        └─────┬──────┘
-           └─────────────────┴────────────────────┘
+           │                 ├────────────────────┐──────────────────┐
+           │          ┌──────┴─────┐        ┌─────┴──────┐     ┌─────┴──────┐
+           │          │ TempestaFW │        │ Web Server │     |   Client   |
+           │          └──────┬─────┘        └─────┬──────┘     └─────┬──────┘
+           └─────────────────┴────────────────────┴──────────────────┘
               Separated network for test traffic
 ```
 
-There is two different models of tests: workload tests and pure functional
-tests. Workload tests uses fully functional HTTP benchmark programs (ab,
-wrk) and HTTP servers (Apache, nginx) to check TempestaFW behaviour. This type
+There is three different models of tests: workload tests (deprecated),
+pure functional tests (deprecated) and user configured tests.
+Workload tests uses fully functional HTTP benchmark programs (ab, wrk) and
+HTTP servers (Apache, nginx) to check TempestaFW behaviour. This type
 of tests is used for schedulers, stress and performance testing.
 
 Pure functional tests check internal logic. Here combined HTTP client-server
 server is used. It sends HTTP messages to TempestaFW, analyses how they are
 forwarded to server, and vice versa, which server connections are used.
 
+User configured tests allow user to write thei own tests with using high level
+primitives, such as different types of clients and servers. Each test contains
+2 parts: declaration of clients and servers and code of test, where this items
+are started, stopped, requests performed, etc. Declarative description of tests
+allows modify items without changing tests, because constructors are called by
+framework instead of test.
```

Comment:
Since workload tests are deprecated, it's unclear whether the new model with configured (acrually "configurable") tests allow using wrk or ab for high performance testing?

Reply:
fixed

Reply:
fixed